packages/helper-grammar-regex-collection/test.js

```diff
@@ -459,9 +459,20 @@ function fixturesIterator(fixturesList, next) {
   });
 }
 
+function addModifiedLines(valid) {
+  const [text, expected] = valid[0];
+  const diffLines = [[`-${text}`, expected], [`+${text}`, expected]];
+
+  return [].concat([], diffLines, valid);
+}
```

Comment:
This automatically adds test for deletion and additions. I hope no one will be confused by this, but it feels worng to me to add those to every single test manually.